torchaudio_contrib/functional.py

```diff
@@ -33,6 +76,15 @@ def stft(signal, fft_len, hop_len, window,
 
     # (!) Only 3D, 4D, 5D padding with non-constant
     # padding are supported for now.
+
+    if signal.dim() == 2:
+        # This is added because otherwise F.pad does not work.
```

Comment:
I thought we were going to move away from adding explicit padding and letting `torch.stft` handle it. We talked about it [here](https://github.com/keunwoochoi/torchaudio-contrib/pull/34#discussion_r271429945) and [here](https://github.com/keunwoochoi/torchaudio-contrib/pull/34#discussion_r272166166). Is there a strong incentive to keep it?

Reply:
Oh, yes this is more like a temporary-fix-meanwhile.

Reply:
Ok fair enough then :)